gobblin-cluster/src/test/java/gobblin/cluster/GobblinClusterKillTest.java

```diff
@@ -107,9 +108,11 @@ private void setupTestDir() throws IOException {
    */
   private void setupManager(int id) throws Exception {
     _clusterManagers[id] =
-        new GobblinClusterManager(TestHelper.TEST_APPLICATION_NAME, TestHelper.TEST_APPLICATION_ID,
-            _config.withValue(GobblinClusterConfigurationKeys.HELIX_INSTANCE_NAME_KEY,
-                ConfigValueFactory.fromAnyRef("Manager_" + id)),
+        new GobblinClusterManager(TestHelper.TEST_APPLICATION_NAME, TestHelper.TEST_APPLICATION_ID, _config
+                .withValue(GobblinClusterConfigurationKeys.HELIX_INSTANCE_NAME_KEY,
+                        ConfigValueFactory.fromAnyRef("Manager_" + id))
+                .withValue(ConfigurationKeys.STATE_STORE_ROOT_DIR_KEY,
```

Comment:
@htran1 @ibuenros @sahilTakiar StateStoreCleaner desperately needs to support all state store types.  Once it does, we can add this back.